.eslintrc.js

```diff
@@ -0,0 +1,103 @@
+//Emma Luk
+
+module.exports = {
+    "env": {
+        "browser": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "indent": [
+            "error",
+            4
+        ],
+        "linebreak-style": [
+            "error",
+            "windows"
+        ],
+        "quotes": [
+            "error",
+            "single"
+        ],
+        "semi": [
+            "error",
+            "always"
+        ],
+
+        // disallow trailing whitespace at the end of lines
+        "no-trailing-spaces": [0],
+
+        // enforces spacing between keys and values in object
+        "key-spacing": [0],
+
+        // disallow irregular whitespace outside of strings and comments
+        "no-irregular-whitespace": [0],
+
+        // disallow use of undefined when initializing variables
+        "no-undef-init": [0],
+
+        // disallow use of undeclared variables
+        "no-undef": [0],
+
+        // specify whether double or single quotes should be used
+        "quotes": [0],
+
+        // disallow use of console (off by default in the node environment)
+        "no-console": [0],
+
+        // disallow trailing commas in object literals
+        "no-comma-dangle": [0],
```

Comment:
This should be 2 or "error" and a second configuration value of "never" is required for this rule.